libr/core/rtr_http.c

```diff
@@ -23,10 +23,8 @@ static int r_core_rtr_http_run(RCore *core, int launch, int browse, const char *
 
 	if (!r_file_is_directory (root)) {
 		if (!r_file_is_directory (homeroot)) {
-			eprintf ("Cannot find http.root (%s) or http.homeroot (%s)\n", root, homeroot);
-			return false;
+			eprintf ("Cannot find http.root or http.homeroot\n");
```

Comment:
can this function run even withoutn http.root/homeroot?

Reply:
the webserver apparently has a REST api